src/webWidget/models.ts

```diff
@@ -189,9 +192,15 @@ export type WebWidgetSignResult = {
   transactionId?: string
 }
 
+/** Result for Key Export action */
+export type WebWidgetKeyExportResult = {
+  //  no result - just success or error
```

Comment:
should there be a `success` boolean?

Reply:
The other methods always return something, in this case, there is no return. The user may, for example, only want to see the keys and not delete them.

And even when deleting, there is nothing to report to the application.
In the webwidget, whenever the user closes the widget, a "user closed" error is returned, but in this case it could be that the user just finished everything he wanted

Reply:
I see, so the success state is just the absence of an error. Ok, that works for me.